infoschema/tables.go

```diff
@@ -1131,14 +1134,25 @@ var tableInspectionResultCols = []columnInfo{
 	{"DETAILS", mysql.TypeVarchar, 256, 0, nil, nil},
 }
 
+var tableMetricTablesCols = []columnInfo{
+	{"METRIC_NAME", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"PROMQL", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"GENERATE_PROMQL", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
Prefer to remove this column, which redundant with the `PROMQL + QUANTILE`.

Reply:
great,done.